crates/bevy_pbr/src/meshlet/material_pipeline_prepare.rs

```diff
@@ -195,7 +192,10 @@ pub fn prepare_material_meshlet_meshes_main_opaque_pass<M: Material>(
                 ],
                 push_constant_ranges: vec![],
                 vertex: VertexState {
-                    shader: MESHLET_MESH_MATERIAL_SHADER_HANDLE,
+                    shader: load_embedded_asset!(
```

Comment:
Can we extract the handles for this shader to the top of the function, and just clone it each time?

Reply:
Same for the other two places please!